### DISCUSSION TITLE
Is the 30$-40$/month revenue per month available for payout after a month?

### DISCUSSION THREAD

**Discussion by ghost** [OlivierGaland/CashFactory#32] [Q&A]
Each apps have different minimum payout rates, so I was having a question: Can I get the as-advertised 30-40 dollars per month without paying for each apps to have enough money for payout, given that I let all the applications run 24/7 at all times and never ever taking a break? And if I can't, how much would I get after the first month?

**Comment by Biocef** [comment]
For me it's 30 dollars ... in 7 months (24/7), all apps includes, don't waste your time.
